python/setup.cfg

```diff
@@ -11,9 +11,6 @@ license_files = LICENSE
 classifiers =
     Programming Language :: Python :: 3
     Programming Language :: Python :: 3 :: Only
-    Programming Language :: Python :: 3.6
-    Programming Language :: Python :: 3.7
-    Programming Language :: Python :: 3.8
```

Comment:
Should we bump python_requires below too?

Reply:
https://github.com/bufbuild/protoc-gen-validate/pull/1225/commits/d8495617d908aad2df19b8b6746bc57b0d281e92